src/lms-harmonizer/tests_integration_sql/procs/mssql/harmonize_assignments/test_lmsassignments.py

```diff
@@ -0,0 +1,121 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+from sqlalchemy.engine.base import Connection
+from tests_integration_sql.mssql_helper import (
+    script_sql,
+    insert_lms_section,
+    insert_lms_section_deleted,
+    insert_edfi_section,
+)
+
+SOURCE_SYSTEM = "Test_LMS"
+PROC_SQL_DEFINITION = script_sql("1080-lms-assignment.sql")
+PROC_EXEC_STATEMENT = "EXEC lms.harmonize_assignment;"
+
+
+def describe_when_lms_and_ods_tables_are_both_empty():
+    def it_should_run_successfully(test_mssql_db: Connection):
+        # arrange
+        test_mssql_db.execute(PROC_SQL_DEFINITION)
+
+        # act
+        test_mssql_db.execute(PROC_EXEC_STATEMENT)
+        # assert - no errors
+
+
+# def describe_when_lms_and_ods_tables_have_no_matches():
```

Comment:
What is the plan for this commented-out code?

Reply:
It is a copy of another test, I am using it as a guide to create my tests, currently working on it